### ISSUE TITLE
StringLiteral should contain information about the type of quotes used (single or double)

### ISSUE THREAD

**Issue by Maximaximum** [dsherret/ts-simple-ast#152]
It would also be great to let the caller change the quotes type for a literal.

**Comment by Maximaximum** [comment]
Related to https://github.com/dsherret/ts-simple-ast/issues/150

**Comment by dsherret** [comment]
Good idea! Maybe something like the following? Or something else?

```typescript
enum QuoteType {
    Single,
    Double
}

// then this method in StringLiteral.ts
getQuoteType(): QuoteType;
```

This is an easy feature to implement. I believe it's just looking at what the first character is: `this.getText()[0] === `"``. If you would like to submit a pull request feel free, otherwise I will get to this eventually.

**Comment by Maximaximum** [comment]
On the other hand, there is already a similar enum - `StringChar`: https://github.com/dsherret/ts-simple-ast/blob/41a9991b0eb663eec367e303e8a6a667f1fef88b/src/ManipulationSettings.ts#L5 You might find it reasonable to use it instead of creating another one.

**Comment by dsherret** [comment]
Renamed `StringChar` to `QuoteType`. Added `.getQuoteType()` on `StringLiteral`.

Implemented in v2.0.0.
